README.md

```diff
@@ -61,12 +61,19 @@ scripts/generate-secret.sh
 ```
 
 #### Manually
+Copy [releases/secret.yml.tmpl](https://github.com/digitalocean/digitalocean-cloud-controller-manager/blob/master/releases/secret.yml#L10) to releases/secret.yml and replace the placeholder in the copy with your base64 encoded token.
```

Comment:
I think this part should be after the bit about base64 encoding the token since we refer to the base64 encoded token and it may be confusing to what that actually is for people.